tools/slack/guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Slack guidelines
+---
+
+[_Back to Slack page_](../)
+
+Slack is a chat client that provides us with a centralized way to communicate without overloading our inboxes. You can [login online](https://gsa-tts.slack.com/) or download the [desktop](https://itunes.apple.com/us/app/slack/id803453959?mt=12) or [mobile](https://itunes.apple.com/us/app/slack-team-communication/id618783545?mt=8) clients from the App Store.
+
+See the [Slack @ TTS](https://docs.google.com/document/d/1Hm42cg61S7FPhaLrRIJxl-LXQCcwGvJTKX_wG0Jz4aU/edit#heading=h.ho6etfp5g3pp) guide to learn about our unique uses, bots and channels.
+
+## Channels
+
+- Teams
+  - 18F - [#news-18f](https://gsa-tts.slack.com/messages/news-18f/) for all 18F-level announcements
+  - [OPP](https://gsa-tts.slack.com/messages/opp/)
+  - [PIFs](https://gsa-tts.slack.com/messages/pifs/)
+  - [Office of Acquisition](https://gsa-tts.slack.com/messages/tts-oa/)
+- Locations
+  - [chitown](https://gsa-tts.slack.com/messages/chitown/)
+  - [dc](https://gsa-tts.slack.com/messages/dc/)
+  - [nyc](https://gsa-tts.slack.com/messages/nyc/)
+  - [sf](https://gsa-tts.slack.com/messages/sf/)
+  - [distributed](https://gsa-tts.slack.com/messages/distributed/)
+
+See a list of other popular channels in the [Slack @ TTS](https://docs.google.com/document/d/1Hm42cg61S7FPhaLrRIJxl-LXQCcwGvJTKX_wG0Jz4aU/edit#heading=h.b0dsxkh3r8wi) guide.
+
+## How to make an announcement in #news
+
+Use #news for vital team announcements. [#news](https://gsa-tts.slack.com/archives/news) is an announcement-only channel where only administrators can post. If you have a post you'd like to go up in #news:
+
+- **18F employees** ping @commsquad or post in [#outreach](https://gsa-tts.slack.com/archives/news) to get a review of the language. Once it's ready, Dahianna Salazar Foreman can post it.
```

Comment:
Understand this is old content but this contacts have changed and we should be using slack channels or google groups generally for request workflows